packages/server/graphql/intranetSchema/mutations/updateEmail.ts

```diff
@@ -0,0 +1,74 @@
+import {GraphQLBoolean, GraphQLNonNull, GraphQLString} from 'graphql'
+import getRethink from '../../../database/rethinkDriver'
+import {requireSU} from '../../../utils/authorization'
+import {InternalContext} from '../../graphql'
+import updateUser from '../../../postgres/queries/updateUser'
+
+const updateEmail = {
+  type: GraphQLNonNull(GraphQLBoolean),
+  description: `Updates the user email`,
+  args: {
+    oldEmail: {
+      type: GraphQLNonNull(GraphQLString),
+      description: 'User current email'
+    },
+    newEmail: {
+      type: GraphQLNonNull(GraphQLString),
+      description: 'User new email'
+    }
+  },
+  resolve: async (_source, {oldEmail, newEmail}, {authToken}: InternalContext) => {
+    const r = await getRethink()
+
+    // AUTH
+    requireSU(authToken)
+
+    // VALIDATION
+    if (!newEmail) {
+      throw new Error('New email can not be empty')
+    }
+
+    if (oldEmail === newEmail) {
+      throw new Error('New email is the same as the old one')
+    }
+
+    const email = newEmail.toLowerCase().trim()
```

Comment:
let's normalize both emails before the validation begins

Reply:
Fixed by using `GraphQLEmailType` which doesn't allow spaces and makes the value lowercase.